Automattic/wp-calypso
# Prevent JS error in pricing page for subdirs sites

### Changes proposed in this Pull Request

This PR fixes a JS issue that occurs when visiting the pricing page in cloud with a secondary site of a multisite network (with sub directories). See capture below.

### Testing instructions

- Run this PR
- Make sure you have a secondary site available (from a multisite on subdirs configuration). Ping me if you need details about how to set one.
- Open your browser console and visit `http://jetpack.cloud.localhost:3000/pricing/?site=:slug&source=jetpack-plans`. `:slug` should be something like `example.com::other-site`.
- Check that you don't see any error in the console

### Screenshots

<img width="409" alt="Screen Shot 2021-01-13 at 2 50 28 PM" src="https://user-images.githubusercontent.com/1620183/104607444-d40d2080-564e-11eb-9d88-98b03f30bdc3.png">
